### DISCUSSION TITLE
IOTA-Lottery

### DISCUSSION THREAD

**Discussion by Ossy63** [iotaledger/firefly#8040] [Help & Support]
Hi, is this real? "You won 2000 IOTA - www.iotalottery.com" 
This message came on my Firefly _ Wallet.
Thx for help!
Greetings, Ossy.

**Comment by begonaalvarezd** [comment]
⚠️ Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
In your wallet go to:
1) Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
3) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/7195b5f8-699b-45d9-b361-26255f610a18)

**Reply by Ossy63** [reply]
Thx for prompt answer! This was also my thinking....
Greetings and thanks!
